clean.cmd

```diff
@@ -0,0 +1,11 @@
+@echo off
+
+REM Copyright (c) .NET Foundation and contributors. All rights reserved.
+REM Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+if "%~1"=="" (

```

Comment:
I figured that this would be the least-destructive way to onboard the commands into the CLI repo since the default command would be to remove the output of the build; if someone executed that accidentally, it would take another ~30 minutes at least to rebuild the repo. I'm open to changing it if you feel strongly one way or another :)